receiver/prometheusreceiver/internal/otlp_transaction_test.go

```diff
@@ -14,125 +14,125 @@
 
 package internal
 
-import (
-	"context"
-	"testing"
-	"time"
+// import (
+// 	"context"
+// 	"testing"
+// 	"time"
 
-	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/model/labels"
-	"github.com/prometheus/prometheus/scrape"
-	"github.com/stretchr/testify/require"
-	"go.opentelemetry.io/collector/component/componenttest"
-	"go.opentelemetry.io/collector/config"
-	"go.opentelemetry.io/collector/consumer/consumertest"
-)
+// 	"github.com/prometheus/common/model"
+// 	"github.com/prometheus/prometheus/model/labels"
+// 	"github.com/prometheus/prometheus/scrape"
+// 	"github.com/stretchr/testify/require"
+// 	"go.opentelemetry.io/collector/component/componenttest"
+// 	"go.opentelemetry.io/collector/config"
+// 	"go.opentelemetry.io/collector/consumer/consumertest"
+// )
 
-func Test_transaction_pdata(t *testing.T) {
-	// discoveredLabels contain labels prior to any processing
-	discoveredLabels := labels.New(
-		labels.Label{
-			Name:  model.AddressLabel,
-			Value: "address:8080",
-		},
-		labels.Label{
-			Name:  model.MetricNameLabel,
-			Value: "foo",
-		},
-		labels.Label{
-			Name:  model.SchemeLabel,
-			Value: "http",
-		},
-	)
-	// processedLabels contain label values after processing (e.g. relabeling)
-	processedLabels := labels.New(
-		labels.Label{
-			Name:  model.InstanceLabel,
-			Value: "localhost:8080",
-		},
-	)
+// func Test_transaction_pdata(t *testing.T) {
```

Comment:
Why this (and other) tests are commented?

Reply:
hi, sorry, missed this somehow. Fixed it now!